BraveShared/PaymentRequest/PaymentRequest.swift

```diff
@@ -2,43 +2,51 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-struct PaymentRequest: Decodable {
-    struct MethodData: Decodable {
-        struct SupportedInstrumentsData: Decodable {
-            let supportedNetworks: [String]
-            let supportedTypes: [String]?
+public enum PaymentRequestResponse {
+    /// The user cancelled the payment
+    case cancelled
+    /// The user successfully completed payment and received an order resposne
+    /// from the server
+    case completed(_ response: String)
+}
+
+public struct PaymentRequest: Decodable {
+    public struct MethodData: Decodable {
+        public struct SupportedInstrumentsData: Decodable {
+            public let supportedNetworks: [String]
```

Comment:
I don't think we need all the members to be public. Let me know if I'm missing something.

Reply:
Everything has to be public because it is in a separate framework (`BraveShared`) otherwise it is not accessible where its used in `Client`